### PR TITLE
[Docs] PR for Add necessary packages for airgapped installation

### PR DESCRIPTION
**Pull request by lizayugabyte**

### CODE REVIEW

File: docs/content/preview/yugabyte-platform/troubleshoot/universe-issues.md [hunk 1]
```diff
@@ -284,3 +284,22 @@ Mem: 251G
...
+
+If you disable **Assign Public IP** during the universe creation, the process may fail under certain conditions, unless you either install the following packages on the Amazon Linux (AMI) or make them available on an accessible package repository:
+
+- `chrony`, if you enabled **Use Time Sync** for the selected cloud provider.
+- `python-minimal`, if YugabyteDB Anywhere is installed on Ubuntu 18.04.
+- `python-setuptools`, if YugabyteDB Anywhere is installed on Ubuntu 18.04.
+- `python-six` or `python2-six` (the Python2 version of Six).
+- `policycoreutils-python`, if YugabyteDB Anywhere is installed on CentOS 7 or Oracle Linux 8.
+- `selinux-policy` must be on an accessible package repository, if YugabyteDB Anywhere is installed on Oracle Linux 8.
+- locales, if YugabyteDB Anywhere is installed on Ubuntu.
```
Comment (favre49) [inline docs/content/preview/yugabyte-platform/troubleshoot/universe-issues.md:301]:
```suggestion
- `locales`, if YugabyteDB Anywhere is installed on Ubuntu.
```

File: docs/content/preview/yugabyte-platform/troubleshoot/universe-issues.md [hunk 2]
```diff
@@ -284,3 +284,22 @@ Mem: 251G
 Disk: /dev/sda2       208G  5.1G  203G   3% /
 ```
 
+## Troubleshoot universe creation
+
+You typically create universes by navigating to **Universes > Create universe > Primary cluster**, as per the following illustration:
+
+![Troubleshoot universe](/images/yb-platform/troubleshoot-uni-creation.png)<br>
+
+If you disable **Assign Public IP** during the universe creation, the process may fail under certain conditions, unless you either install the following packages on the Amazon Linux (AMI) or make them available on an accessible package repository:
```
Comment (favre49) [inline docs/content/preview/yugabyte-platform/troubleshoot/universe-issues.md:293]:
```suggestion
If you disable **Assign Public IP** during the universe creation, the process may fail under certain conditions, unless you either install the following packages on the machine image or make them available on an accessible package repository:
```

File: docs/content/preview/yugabyte-platform/troubleshoot/universe-issues.md [hunk 3]
```diff
@@ -284,3 +284,22 @@ Mem: 251G
...
+- `python-minimal`, if YugabyteDB Anywhere is installed on Ubuntu 18.04.
+- `python-setuptools`, if YugabyteDB Anywhere is installed on Ubuntu 18.04.
+- `python-six` or `python2-six` (the Python2 version of Six).
+- `policycoreutils-python`, if YugabyteDB Anywhere is installed on CentOS 7 or Oracle Linux 8.
+- `selinux-policy` must be on an accessible package repository, if YugabyteDB Anywhere is installed on Oracle Linux 8.
+- locales, if YugabyteDB Anywhere is installed on Ubuntu.
+
+The preceding package requirements are applicable to YugabyteDB Anywhere version 2.13.1.0.
+
+If you are using YugabyteDB Anywhere version 2.12.*n*.*n* and disable **Use Time Sync** during the universe creation, you also need to install the `ntpd` package.
```
Comment (favre49) [inline docs/content/preview/yugabyte-platform/troubleshoot/universe-issues.md:305]: 2.12.n.n means "any 2.12 version" right?

**Review by favre49 — APPROVED** [review]
Small suggestions, LGTM besides that.

File: docs/content/preview/yugabyte-platform/troubleshoot/universe-issues.md (hunk 3 above)
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/troubleshoot/universe-issues.md:305]: Yes

File: docs/content/preview/yugabyte-platform/troubleshoot/universe-issues.md (hunk 2 above)
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/troubleshoot/universe-issues.md:293]: Changed

File: docs/content/preview/yugabyte-platform/troubleshoot/universe-issues.md (hunk 1 above)
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/troubleshoot/universe-issues.md:301]: Applied change

**Pull request merged**